### ISSUE TITLE
JS error occurs when pinching off screen 

### ISSUE THREAD

**Issue by scottyantipa** [transcriptic/react-map-interaction#20]
This doesn't appear to break any functionality, but it is annoying.

To repro:
- Open in simulator or on a mobile device
- pinch and zoom, then bring one of those fingers onto the edge of the device so it just touches 
- note the js error

This also may have to do with the "pull back to go back in history" default behavior of the browser.

![undefined-object-bug](https://user-images.githubusercontent.com/3643611/42064741-51f951f8-7aec-11e8-9366-357a851232aa.png)

**Comment by tpason** [comment]
let try
in react-map-interaction.js file
```
key: 'onDrag',
    value: function onDrag (pointer) {
      if (typeof this.startPointerInfo !== "undefined" && typeof this.startPointerInfo.pointers[0] === "undefined") {
        return;
      }
```
It's fixed for me :D

**Comment by wallzero** [comment]
Could this error be looked into? On my page all touch input is borked once this error is thrown - I can't scroll, click buttons, etc.

**Comment by scottyantipa** [comment]
Sure I'll fix it

**Comment by scottyantipa** [comment]
@wallzero @tpason This was fixed and released in v1.3.1. Let me know how it works out.
